### ISSUE TITLE
Allow eleventyExcludeFromCollections to take an array of collections

### ISSUE THREAD

**Issue by paulrobertlloyd** [11ty/eleventy#1175]
**Is your feature request related to a problem? Please describe.**
I have an index file alongside a group of posts

```
posts
├─ 01-01-2020-first-post.md
├─ 02-02-2020-second-post.md
├─ index.njk
└─ posts.11tydata.js
```

`posts.11tydata.js` adds a `post` tag to all posts in this folder, and `index.njk` loops through `collection.post`, but this loop includes `index.njk` as well as all the individual posts.

I can exclude the index by using `eleventyExcludeFromCollections: true`, but that excludes it from all collections. But in my `sitemap.xml` I want to include it when I use `collection.all`.

**Describe the solution you'd like**
Allow `eleventyExcludeFromCollections` to take an array of collections to be excluded from:

```yaml
eleventyExcludeFromCollections:
- post
- foo
- bar
```

**Describe alternatives you've considered**

Currently I add `tags: false` to the index file, but it’s not as explicit.

**Comment by zachleat** [comment]
I like this idea! It must go into the queue!

> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by davidhund** [comment]
Apologies if commenting here (and now) is inappropriate. 
I'm not entirely sure how the "closed-but-queued issues" thing works. ;-)

Anyway: this gets a strong 👍  from me: Paul's use-case is something I also run into all the time…

**Comment by Ryuno-Ki** [comment]
@davidhund Basically, Zach's looking at https://github.com/11ty/eleventy#major-roadmapped-features for making decisions.

**Comment by architchandra** [comment]
Please add the ability to exclude from collections on a per-collection basis (or include in `collections.all` while excluding from others). It will be really helpful.

**Comment by nitin2953** [comment]
Please reopen this issue 😥,
without this feature my directory structure is messing up 😭.

**Comment by zachleat** [comment]
#2482 was merged—this will ship with 3.0.

**Comment by zachleat** [comment]
Docs shipping to https://www.11ty.dev/docs/collections/#how-to-exclude-content-from-collections
